frontend/public/components/filter-toolbar.tsx

```diff
@@ -287,6 +349,9 @@ export const FilterToolbar: React.FC<FilterToolbarProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [rowFiltersInitialized, labelSelectionInitialized]);
 
+  const showSearchFilters = Object.keys(filterDropdownItems)?.length !== 0;
+
+  const showSearchFiltersDropdown = Object.keys(filterDropdownItems)?.length > 1;
```

Comment:
Nit: `Object.keys` should always return an array object instance.

```suggestion
  const showSearchFiltersDropdown = Object.keys(filterDropdownItems).length > 1;
```